### ISSUE TITLE
Patch 11.0.2 Error

### ISSUE THREAD

**Issue by Talyrius** [glassleo/GnomishInboxShrinker#4]
I'm receiving the following error on Retail after the release of Patch 11.0.2.

```
86x bad argument #1 to '?' (Usage: SetPortraitToTexture(texture, asset))
[string "=[C]"]: in function `SetPortraitToTexture'
[string "@GnomishInboxShrinker/GnomishInboxShrinker.lua"]:291: in function `Update'
[string "@GnomishInboxShrinker/GnomishInboxShrinker.lua"]:162: in function `UpdateInboxScroll'
[string "@GnomishInboxShrinker/GnomishInboxShrinker.lua"]:245: in function <GnomishInboxShrinker/GnomishInboxShrinker.lua:228>

Locals:
(*temporary) = "bad argument #1 to '?' (Usage: SetPortraitToTexture(texture, asset))"
```

**Comment by glassleo** [comment]
Having trouble reproducing this, what did you do when you got the error?

**Comment by glassleo** [comment]
Fixed with e69c853
